### PR TITLE
[ko] Clear some IsSet warnings for Hugo build

### PR DESCRIPTION
**Pull request by Sea-n**
Adapt small change from PR #35813.
- #35813

Checked with `lsync.sh` script.
```bash
./scripts/lsync.sh content/ko/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md
```

### CODE REVIEW

File: content/ko/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ card:
 
 <img src="/images/kubeadm-stacked-color.png" align="right" width="150px"></img>
 이 페이지에서는 `kubeadm` 툴박스 설치 방법을 보여준다.
-이 설치 프로세스를 수행한 후 kubeadm으로 클러스터를 만드는 방법에 대한 자세한 내용은 [kubeadm을 사용하여 클러스터 생성하기](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/) 페이지를 참고한다.
+이 설치 프로세스를 수행한 후 kubeadm으로 클러스터를 만드는 방법에 대한 자세한 내용은 [kubeadm 으로 클러스터 생성](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/) 페이지를 참고한다.
```
Comment (yoonian) [inline content/ko/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:15]:
```suggestion
이 설치 프로세스를 수행한 후 kubeadm으로 클러스터를 만드는 방법에 대한 자세한 내용은 [kubeadm 으로 클러스터 생성하기](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/) 페이지를 참고한다.
```
I think `creating` meant not only `creation` but also action. So I leave comment.

Comment (Sea-n) [reply content/ko/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:15]: Thanks! Update.

(As you know, I am not a Korean speaker. 😅)

**Comment by seokho-son** [comment]
Hi @Sea-n 
It is nice enhancement !! :)

But, Korean team are using our development branch.
Could you let Korean contributors handle this issue ? If you wish to resolve your self using this PR, we can guide you how to use current Korean development branch which is dev-1.24-ko.3. :)

**Comment by Sea-n** [comment]
Thanks @seokho-son for letting me know the right way, I've changed the base branch.

And congrats for finally merged `dev-1.24-ko.2` two days ago! 🎉

**Comment by yoonian** [comment]
@Sea-n how about dev-1.24-ko.3, not dev-1.24-ko.2?

**Comment by Sea-n** [comment]
Thanks @yoonian for jump in and help so quickly. 😅

How about now? I think the git problem should be solved.

**Pull request merged**